package.json

```diff
@@ -1,21 +1,30 @@
 {
   "name": "live-jazz",
-  "version": "0.0.0",
+  "version": "0.1.0",
   "private": true,
   "scripts": {
     "start": "node ./server/bin/www",
     "server": "nodemon ./server/bin/www",
-    "test": "yarn lint && mocha --recursive server/test",
+    "test": "yarn lint && mocha --recursive server/test && CI=true react-scripts test --env=jsdom",
+    "dev:server": "react-scripts start",
```

Comment:
Let's rename `yarn dev:server` → `yarn frontend` and `yarn server` → `yarn backend`.